site/content/en/reference/live/migrate/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Migrate"
+linkTitle: "migrate"
+type: docs
+description: >
+   Migrate the package inventory object to a ResourceGroup custom resource
+---
+<!--mdtogo:Short
+    Migrate the package inventory object to a ResourceGroup custom resource
+-->
+
+NOTE: This command is not available unless the RESOURCE_GROUP_INVENTORY
+environment variable is set.
+
+The migrate command upgrades the inventory object from a ConfigMap to
+a ResourceGroup custom resource. The migrate performs the following steps:
+
+1. Applies the ResourceGroup custom resource definition (see
+   `kpt live install-resource-group`)
+2. If a ConfigMap inventory object exists in the cluster, the inventory
```

Comment:
Can you add some note that the ResourceGroup CR from the `migrate` command doesn't have the same name or inventory-id as the ConfigMap inventory object.